manifest.json

```diff
@@ -163,6 +163,12 @@
             "description": "Help the store to stay responsive and adjust to the user’s device capabilities and network speed.",
             "type": "boolean",
             "default": true
+          },
+          "enableFiltersFetchOptimization": {
+            "title": "Enables fetching filters partially",
+            "description": "Improves search pages' performance by fetching only the 10 first facets of every filter, adding a button to show the rest.",
+            "type": "boolean",
+            "default": false
```

Comment:
I think we should publish this as `false` first, then run the script the saves the settings, then create another PR changing the default to `true`, right?

Reply:
Yes, but since this changes the filters' layout, I am afraid to set it as true 👀